### PR TITLE
Lottery Ticket Powernerf

### PR DESCRIPTION
**Pull request by BurgerLUA**
# Overview
Nerfs lottery tickets. I did the math, and the average earnings of a lottery ticket is 4000 credits each. You can earn a shitton of credits through lottery tickets purchased through mining and it felt super ridiculous.

## Balance
Here are the percentages for lottery tickets for this PR.
50% - Nothing
30% - 200 credits.
10% - 500 credits.
5% - 1,000 credits.
2.5% - 2,500 credits
1.5% - 5,000 credits
0.5% - 10,000 credits
0.25% - 25,000 credits.
0.25% - 50,000 credits.

## Ways to Obtain
Lottery tickets can now be purchased from cigarette machines for 200 credits. Of course, they can still be traded at mining.

### CODE REVIEW

**Comment by ParadoxSpace** [comment]
nty it's pretty balanced as is.

File: html/changelogs/sdtwbaj-skrellmoney.yml [hunk 1]
```diff
@@ -1,37 +1,37 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Skrell have a higher chance to get more money."
```
Comment (Arrow768) [inline html/changelogs/sdtwbaj-skrellmoney.yml]: Hm, that looks like the linebreaks got converted from linux to windows or the other way round.

Comment (BurgerLUA) [reply html/changelogs/sdtwbaj-skrellmoney.yml]: I'm on windows 10 because I'm not a hipster.

**Comment by ParadoxSpace** [comment]
>work in an incredibly dangerous job, IRL and IG, to where people routinely joke
>these people are well-compensated for their efforts
>somehow this is a problem
Again, there is no tangible downside or negative effect to miners having a bunch of money. Urist McShaftMiner having $40k effects the station in no negative fashion. This really should be discussed somewhere.

**Comment by Arrow768** [comment]
Since economy is still planned, we shouldn't have items that magically produce money.
There is definitely something off if a shaft miner suddenly has more money than all of the departments together.

**Comment by BurgerLUA** [comment]
It's important to note that the lotto tickets in this PR are still generous with money, but not as much as they were previously. You have a 50% chance of losing, 30% chance of breaking even, and a 20% chance of gaining more money.

**Comment by LordFowl** [comment]
>economy
>planned
tomcruiselaugh.jpg

File: html/changelogs/BurgerBB-Lottery.yml [hunk 2]
```diff
@@ -0,0 +1,6 @@
+author: BurgerBB
+ 
+delete-after: True
+
+changes: 
+  - balance: "Rebalanced lottery tickets so you don't win as much. Lottery tickets can now be purchased at cigarretee machines as well."
```
Comment (Arrow768) [inline html/changelogs/BurgerBB-Lottery.yml]: spelling mistake at cigarretee

**Pull request merged**